democracy/views/section_comment.py

```diff
@@ -42,6 +42,10 @@ def to_internal_value(self, data):
             data["plugin_data"] = ""
         if data.get("images") is None:
             data["images"] = []
+        user = self.context['request'].user
+        if user and data.get('author_name'):
+            user.nickname = data['author_name']
+            user.save()
```

Comment:
Also, IMO `to_internal_value()` seems a bit wrong place for this, maybe `save()` ?